.github/workflows/maven.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Setup Java
         uses: actions/setup-java@v2
         with:
-          distribution: 'adopt'
+          distribution: 'zulu'
```

Comment:
I don't think I want to test with Zulu since it's not what most users use.

Reply:
the reason that I removed 'adopt' is because it is no longer maintained.

https://github.com/actions/setup-java/blob/main/README.md

```
NOTE: Adopt OpenJDK got moved to Eclipse Temurin and won't be updated anymore. It is highly recommended to migrate workflows from adopt to temurin to keep receiving software and security updates. 
```

I just pushed a change. I added a JDK matrix:
```
jdk: ['temurin', 'zulu']
```

wdyt?

Reply:
Sure, that sounds good.